packages/glue/src/index.ts

```diff
@@ -192,7 +192,8 @@ interface ICoversOptions {
  */
 export class Covers {
   private coverPoints = new Map<number, CoverPoint>();
-  // @ts-ignore
+  // Add this for early returns
+  private ignoredCoverPoints = new Array<number>()
```

Comment:
I don't think we need to keep track of ignored cover points at all. Can't we just return early?

Reply:
Okay, sure. 😀